pom.xml

```diff
@@ -158,6 +158,9 @@
                 <configuration>
                     <source>${maven.compiler.source}</source>
                     <target>${maven.compiler.target}</target>
+                    <excludes>
+                        <exclude>TransactionsExample.java</exclude>
```

Comment:
@programmatix I've excluded the `TransactionsExample.java` because I couldn't get it to compile successfully. 

Generally we should be able to run most of the examples, and on occasions where it's not ideal, they should at least compile. Obviously, If we have a genuine reason for this then sure, just wanted to bring attention to this.

For context, this is the error I encountered when trying to compile it:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.0:compile (default-compile) on project docs-code: Compilation failure: Compilation failure: 
[ERROR] /Users/mariashodunke/Couchbase/docs/docs-sdk-java/modules/howtos/examples/TransactionsExample.java:[22,53] package com.couchbase.client.core.transactions.events does not exist
[ERROR] /Users/mariashodunke/Couchbase/docs/docs-sdk-java/modules/howtos/examples/TransactionsExample.java:[23,53] package com.couchbase.client.core.transactions.events does not exist
[ERROR] /Users/mariashodunke/Couchbase/docs/docs-sdk-java/modules/howtos/examples/TransactionsExample.java:[24,53] package com.couchbase.client.core.transactions.events does not exist
[ERROR] /Users/mariashodunke/Couchbase/docs/docs-sdk-java/modules/howtos/examples/TransactionsExample.java:[25,53] package com.couchbase.client.core.transactions.events does not exist
[ERROR] /Users/mariashodunke/Couchbase/docs/docs-sdk-java/modules/howtos/examples/TransactionsExample.java:[26,50] package com.couchbase.client.core.transactions.log does not exist
[ERROR] /Users/mariashodunke/Couchbase/docs/docs-sdk-java/modules/howtos/examples/TransactionsExample.java:[48,50] package com.couchbase.client.java.transactions.log does not exist
[ERROR] /Users/mariashodunke/Couchbase/docs/docs-sdk-java/modules/howtos/examples/TransactionsExample.java:[435,34] cannot find symbol
[ERROR]   symbol:   class CoreTransactionLogMessage
[ERROR]   location: class TransactionsExample
[ERROR] /Users/mariashodunke/Couchbase/docs/docs-sdk-java/modules/howtos/examples/TransactionsExample.java:[427,34] cannot find symbol
[ERROR]   symbol:   class IllegalDocumentStateEvent
[ERROR]   location: class TransactionsExample
[ERROR] /Users/mariashodunke/Couchbase/docs/docs-sdk-java/modules/howtos/examples/TransactionsExample.java:[429,23] cannot find symbol
[ERROR]   symbol:   class TransactionEvent
[ERROR]   location: class TransactionsExample
[ERROR] /Users/mariashodunke/Couchbase/docs/docs-sdk-java/modules/howtos/examples/TransactionsExample.java:[442,34] cannot find symbol
[ERROR]   symbol:   class TransactionCleanupAttemptEvent
[ERROR]   location: class TransactionsExample
[ERROR] /Users/mariashodunke/Couchbase/docs/docs-sdk-java/modules/howtos/examples/TransactionsExample.java:[442,85] cannot find symbol
[ERROR]   symbol:   class TransactionCleanupEndRunEvent
[ERROR]   location: class TransactionsExample
[ERROR] /Users/mariashodunke/Couchbase/docs/docs-sdk-java/modules/howtos/examples/TransactionsExample.java:[443,23] cannot find symbol
[ERROR]   symbol:   class TransactionEvent
[ERROR]   location: class TransactionsExample
[ERROR] /Users/mariashodunke/Couchbase/docs/docs-sdk-java/modules/howtos/examples/TransactionsExample.java:[649,14] cannot find symbol
[ERROR]   symbol:   class TransactionLogMessage
[ERROR]   location: class TransactionsExample
[ERROR] /Users/mariashodunke/Couchbase/docs/docs-sdk-java/modules/howtos/examples/TransactionsExample.java:[659,14] cannot find symbol
[ERROR]   symbol:   class TransactionLogMessage
[ERROR]   location: class TransactionsExample
[ERROR] -> [Help 1]
```